apps/job-info-consumer/consumer/cmd/utils.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/bacalhau-project/bacalhau/pkg/system"
+	"github.com/multiformats/go-multiaddr"
+	"github.com/spf13/cobra"
+)
+
+func getCommandLineExecutable() string {
+	return os.Args[0]
+}
+
+func getDefaultOptionString(envName string, defaultValue string) string {
+	envValue := os.Getenv(envName)
+	if envValue != "" {
+		return envValue
+	}
+	return defaultValue
+}
+
+func getDefaultOptionInt(envName string, defaultValue int) int {
+	envValue := os.Getenv(envName)
+	if envValue != "" {
+		i, err := strconv.Atoi(envValue)
+		if err == nil {
+			return i
+		}
+	}
+	return defaultValue
+}
+
+func getDefaultOptionBool(envName string, defaultValue bool) bool {
+	envValue := os.Getenv(envName)
+	if envValue != "" {
+		i, err := strconv.ParseBool(envValue)
+		if err == nil {
+			return i
+		}
+	}
+	return defaultValue
+}
```

Comment:
For these you can use 

```
# util.GetEnvAs
# e.g. 
v := util.GetEnvAs[bool]("TEST_BOOL", true, strconv.ParseBool)
v := util.GetEnvAs[int]("TEST_INT", 10, strconv.Atoi)
```

Reply:
Didn't know about this util. Thanks for the pointer Ross :)